node.go

```diff
@@ -86,12 +90,10 @@ func (conn *connection) New(subscripts ...string) (n *node) {
 		panic("YDB: supply node type with at least one string (typically varname)")
 	}
 	n = new(node)
-	n.conn = *conn
+	n.conn = conn
 	n.mutable = false
-	n.bufGo = make([]string, len(subscripts))
+	n.bufGo = subscripts
```

Comment:
This is not strictly necessary, but you can just reference the `subscripts` slice directly, no need to do a 2nd allocate-and-copy here. You're still pinning each string.